src/main/java/com/redhat/cloud/notifications/routers/EndpointService.java

```diff
@@ -222,7 +231,6 @@
             )
     })
     @APIResponse(responseCode = "200", content = @Content(schema = @Schema(type = SchemaType.STRING)))
-    @Produces(MediaType.TEXT_PLAIN)
```

Comment:
The schema is `SchemaType.String` but it says it produces an `APPLICATION_JSON`. Which one is correct?

Reply:
Is that incompatible? I don't know `SchemaType` very well but I don't see any specific value for JSON content and `STRING` seems to be the closest thing available.

`NotificationResources#getNotificationDetails` returns a `JsonObject` instance, that's why I changed the `MediaType` to `APPLICATION_JSON`.

Reply:
It seems we could (or should?) replace with `@Content(schema = @Schema(type = SchemaType.STRING)` with `@Content(mediaType = MediaType.APPLICATION_JSON)` when returning JSON. Now I'm wondering why we would return `SchemaType` at all instead of always return `MediaType` 😄

Reply:
With `SchemaType.STRING`:

![image](https://user-images.githubusercontent.com/10584698/119558023-33b7cd00-bda1-11eb-8c9f-d275aa2f134b.png)

With `MediaType.APPLICATION_JSON`:

![image](https://user-images.githubusercontent.com/10584698/119558210-695cb600-bda1-11eb-94f7-3430103652f0.png)

I guess the second one a bit cleaner but there's no breaking change.

Reply:
> Now I'm wondering why we would return `SchemaType` at all instead of always returning `MediaType`

`SchemaType` helps the openapi creating when we have a custom object; in this case is a string. I'm not sure if it should be plain text. But you are right, in this case application/json is OK vs plaintext.